compiler/dyno/lib/resolution/Resolver.cpp

```diff
@@ -1044,7 +1044,8 @@ void Resolver::resolveTupleUnpackAssign(ResolvedExpression& r,
       actuals.push_back(CallInfoActual(rhsEltType, UniqueString()));
       auto ci = CallInfo (/* name */ USTR("="),
                           /* calledType */ QualifiedType(),
-                          /* isMethod */ false,
+                          /* isMethodCall */ false,
+                          /* isOpCall */ true,
```

Comment:
Since we determine if it's an op call based on the name, I would expect the `CallInfo` constructor to run `isUstrOpName` itself & then we don't need additional arguments here.

Reply:
Whoops, forgot about that usage